app/lib/io/zipUtil.js

```diff
@@ -12,7 +12,26 @@ define([
     var NotImplementedError = errors.NotImplemented;
 
     var unpack = function(async, zipData, io, targetPath){
-        throw new NotImplementedError('ZIP unpack method is not yet implemented');
+        if (async)
+            throw new NotImplementedError('Asynchronous ZIP unpack method is not yet implemented');
+
+        var zip = new JSZip(zipData);
+        var files = zip.files;
+        console.log("unpacking files: ", files);
+        var i;
+
+        for (i in files){
+            var file = files[i];
+            var absolute_path = [targetPath, file.name].join(targetPath[targetPath.length-1]=='/' ? "" : "/");
```

Comment:
We commonly declare names  `var` at the top of the function. and also, you can use just one var for multiple declarations, comma separated.
This is even part of our style guide: https://github.com/metapolator/metapolator/wiki/coding-style#variable-declaration